dependencies.py

```diff
@@ -0,0 +1,12 @@
+from sqlalchemy.ext.asyncio import AsyncSession
+
+from database import SessionLocal
+
+
+async def get_db() -> AsyncSession:
+    db = SessionLocal()
```

Comment:
The 'SessionLocal' is being used to create a new session, but since this is an async context, ensure that 'SessionLocal' is configured for async operations. The current setup seems correct for async usage, but double-check the session configuration in 'database.py' to confirm it aligns with async requirements.